### PR TITLE
Add & reformat int compare functions

### PR DESCRIPTION
**Pull request by gpotter2**
Fixes https://github.com/secdev/scapy/issues/1110

### CODE REVIEW

File: scapy/fields.py [hunk 1]
```diff
@@ -1136,35 +1136,24 @@ def __init__(self, value, names):
...
-        return self.value < self._fixvalue(other)
-    def __le__(self, other):
-        return self.value <= self._fixvalue(other)
-    def __gt__(self, other):
-        return self.value > self._fixvalue(other)
-    def __ge__(self, other):
-        return self.value >= self._fixvalue(other)
-    def __ne__(self, other):
-        return self.value != self._fixvalue(other)
+    __hash__ = lambda self:  hash(self.value)
```
Comment (p-l-) [inline scapy/fields.py]: I don't like using lambda functions here. Is there a good reason for that?

Comment (gpotter2) [reply scapy/fields.py]: Not really. It just takes way less space and is (to me) more readable.

I'm ok to remove them if needed.

Comment (p-l-) [reply scapy/fields.py]: If there is no specific reason for that, then please remove this change.

**Comment by gpotter2** [comment]
@p-l- Updated

**Pull request merged**